docs/tutorials/RemovingFields.adoc

```diff
@@ -0,0 +1,97 @@
+= Tutorial - Removing a field from an entity
+:site-section: DeveloperGuide
+:toc: macro
+:toc-title:
+:imagesDir: ../images/remove
+:stylesDir: ../stylesheets
+:xrefstyle: full
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:warning-caption: :warning:
+:source-highlighter: highlightjs
+endif::[]
+
+[quote, Charles Maturin, Motto to "Bertram"]
+'Tis well to be merry and wise,
+'Tis well to be honest and true;
+'Tis well to be off with the old love,
+Before you are on with the new.
+
+{empty} +
+
+== Introduction
+When working on Address Book, you will most likely find that some features and fields that are no longer necessary.
+In scenarios like this, you can consider refactoring the existing `Person` model to suit your use case.
+
+In this tutorial, we'll do exactly just that and remove the `Address` field from `Person`.
+
+== Safely deleting `Address`
+
+Fortunately, the IntelliJ IDEA provides a robust refactoring tool that can identify _most_ usages.
+Let's try to use it as much as we can.
+
+=== Assisted refactoring
+We start by identifying ``Address``'s usages.
+Select the `Address` class and perfom a `Refactor` > `Safe Delete` through the menu.
+
+.Unsafe delete?
+image::UnsafeDelete.png[width=787px. height=238px]
+{empty} +
+
+Choose to `View Usages` and you should be presented with a list of conflicts.
+
+.Short list of conflicts
+image::SafeDeleteConflicts.png[width=955, height=640px]
+{empty} +
+
+Resolve each conflict by recursively performing ``Safe Delete``s.
+Let's try removing references to `Address` in `EditPersonDescriptor`.
+
+. Safe delete the field `address` in `EditPersonDescriptor`
+. Select `Yes` when prompted to remove getters and setters
+. Select `View Usages` again image:UnsafeDeleteOnField.png[width=1145px, height=583px]
+. Remove the usages of `address` and select `Do refactor` when you are done.
+. Repeat the steps for the remaining usages of `Address`
+
+After you are done, verify that the application still works by compiling and running it again.
+
+=== Manual refactoring
+
+Unfortunately, there are usages of `Address` that IntelliJ IDEA cannot identify. Let's fix them.
```

Comment:
How to find such leftovers? Do a text search? Given we are removing a person field, the reader should be looking through the fxml files anyway, right?

Reply:
We can list places where there could be left-over references e.g., fxml files, test data files, documentation files, ...